webaverse/app
# Rename metaversefile -> totum

ONE OF MANY

metaversefile -> totum

metaversefileApi -> totumApi

.metaversefile -> .totum

All Scenes have been set up to point to ALL forked repositories for testing purposes, this will be reverted to the webaverse repo on merge!

cd packages/totum and checkout hash 8ab8356b36fe8f8eb290c6b2db35cf86bd3bd5f6

QA Checklist:

Test each scene, try out all the items, can you see every single object in the scn list?
Go through FTUX, and interact with everything.



totum repo PR:

https://github.com/webaverse/totum/pull/41
